backend/src/services/enhanced_recommendations.py

```
@@ -99,21 +117,33 @@ def generate_hotel_urls(self, hotel: Dict) -> Dict:
             try:
                 # Format dates for URL parameters
-                ci_date = datetime.strptime(check_in, "%Y-%m-%d").strftime("%Y-%m-%d")
-                co_date = datetime.strptime(check_out, "%Y-%m-%d").strftime("%Y-%m-%d")
+                ci_date = datetime.strptime(check_in, "%Y-%m-%d").strftime(
+                    "%Y-%m-%d"
+                )
+                co_date = datetime.strptime(check_out, "%Y-%m-%d").strftime(
+                    "%Y-%m-%d"
+                )
                 date_params = f"&checkin={ci_date}&checkout={co_date}"
             except:
```

Ruff (0.13.1)

Do not use bare `except`

(E722)